lib/linguist/languages.yml

```diff
@@ -486,6 +486,13 @@ BibTeX:
   codemirror_mode: stex
   codemirror_mime_type: text/x-stex
   language_id: 982188347
+Bicep:
+  type: programming
+  extensions:
+  - ".bicep"
+  tm_scope: source.bicep
+  ace_mode: text
+  language_id: 321200902
```

Comment:
Are you 💯  you don't want to specify a colour? Without specifying a colour, the default `#cccccc` will be used in places like the sidebar and repo lists.

Reply:
Good point, thanks for pointing this one out! I've added a color.